contrib/babelfishpg_common/sql/smalldatetime.sql

```diff
@@ -671,7 +693,7 @@ WITH FUNCTION sys.int2smalldatetime (INT) AS IMPLICIT;
 CREATE OR REPLACE FUNCTION sys.bigint2smalldatetime(IN num BIGINT)
 RETURNS sys.SMALLDATETIME
 AS $$
-    SELECT CAST('1900-01-01 00:00:00' AS sys.SMALLDATETIME) + num;
+    SELECT CAST('1900-01-01 00:00:00' AS sys.SMALLDATETIME) + CAST(num AS INT);
```

Comment:
Could you find or add a test case showing BIGINT to INT has no harm?

Reply:
BIGINT to INT will either convert successfully if INT_MIN < num < INT_MAX, otherwise it will raise an exception for being out of bound for INT. While the error message will be different, ultimately I don't think there is much issue due to the fact that even INT_MAX/INT_MIN is far beyond the acceptable limit of what DATETIME (let alone SMALLDATETIME) can accept. Therefore, Babelfish will raise an error in the same situations as SQL Server, just with an int out of range instead of a datetime out of range error.

Reply:
Added